mhs/common/mhs_common/workflow/asynchronous_express.py

```diff
@@ -10,3 +11,6 @@ class AsynchronousExpressWorkflow(common_asynchronous.CommonAsynchronousWorkflow
     async def handle_outbound_message(self, message_id: str, interaction_details: dict, payload: str) \
             -> Tuple[int, str]:
         raise NotImplementedError()
+
+    async def handle_inbound_message(self, work_description: wd.WorkDescription, payload: str):
```

Comment:
I don't think the ack message actually contains an informational tag. Looking at the ack template it just contains the message id's to associate the ack with the inbound message.

Reply:
I think when I wrote the comment I was imagining that the workflow would construct the ack response and then pass it back to the handler to return to Spine. What you're doing is having the handler construct the ack response instead. Which I'm not so against if the workflows are all returning the same sort of ack back to Spine (in the outbound handler, this isn't the case, so I expect the workflow to determine the response to return back to the supplier system.

Reply:
Yeah I think the inbound handler will pretty much always return the same ack to spine, we're not responding with errors to spine since it doesn't process them, it simply retries. And as mentioned looking over the ack response there's nothing that jumps out that should be variable based on the workflow